gdmix-workflow/images/gdmix/Dockerfile

```diff
@@ -20,21 +23,38 @@ RUN yum  install -y \
     unzip \
     tree \
     curl \
-    python3-devel \
-    python3
+    gcc \
```

Comment:
why gcc is needed?

Reply:
Because Python 3.7.4 has to be compiled from source code, gcc is needed for it.